selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -1620,6 +1638,28 @@ public void waitTextElementsActiveLine(final String expectedText) {
     return getCursorPositionFromWebElement(currentActiveElement);
   }
 
+  public Pair<Integer, Integer> getCurrentCursorPositionFromVisible(int editorIndex) {
```

Comment:
`fromVisible` prefix name looks redundant here so as we are setting editor number.

Reply:
I would also add method's JavaDocs with explanation what kind of information returning object `Pair<Integer, Integer>` holds.